src/rust/sdk/src/lib.rs

```diff
@@ -67,8 +75,8 @@ impl EconiaClient {
     /// # Arguments:
     ///
     /// * `node_url` - Url of aptos node.
-    /// * `econia_address` - Aptos `AccountAddress`.
-    /// * `account` - `LocalAccount` representing Aptos user account.
+    /// * `econia_address` - Aptos [AccountAddress].
+    /// * `account` - [LocalAccount] representing Aptos user account.
```

Comment:
Need to add a destination for structs from external crates for rustdoc to generate a link

Reply:
https://docs.rs/crate/aptos-api/0.2.7

aptos have no docs here...